XenModel/Messages.resx

```diff
@@ -2891,6 +2891,15 @@ Do you want to assign these VMs to the schedule '{0}' instead?</value>
   <data name="COLLAPSE_CHILDREN" xml:space="preserve">
     <value>&amp;Collapse Children</value>
   </data>
+  <data name="COMMAND_RESET_VTPM_WARNING" xml:space="preserve">
+    <value>Resetting the vTPM that is attached to a VM to its original state may result in deactivation of security features on the VM. Do you want to continue?</value>
+  </data>
+  <data name="COMMAND_VTPM_DISABLED_NON_UEFI" xml:space="preserve">
+    <value>TPM is supported only for VMs using the UEFI or UEFI secure boot modes.</value>

```

Comment:
```suggestion
    <value>vTPMs are only supported for VMs using the UEFI or UEFI secure boot modes.</value>
    ```

Reply:
My intention was to differentiate between vTPM as instance of virtual TPM and TPM as a feature - I think in this context we need the latter.